src/components/attendees/AttendeeTable.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import AttendeeRow from './AttendeeRow';
+import { Attendee } from '../../interfaces/Attendee';
+import { ReactComponent as FilterIcon } from '../../assets/Attendees/filter.svg';
+import { ReactComponent as SortIcon } from '../../assets/Attendees/sort.svg';
+import './styles.css';
+import Logistics from '../../pages/logistics';
+
+/**
+ * Creation of the Attendee Table component for the Attendee page. The filter for the Attendee Table will be applied here to display the corresponding hackers.
+ * @returns the table of selected hackathon attendees.
+ */
+export default function AttendeeTable() {
+	const attendeeData = [];
+
+	const [search, setSearch] = useState('');
+	const [namesAscending, setNamesAscending] = useState(false);
+
+	let person1: Attendee = {
+		firstName: 'Jane Jane',
+		lastName: 'Doe',
+		email: 'email@domain.com',
+		discord: 'discord#1234',
+		oAuth: 'email',
+		school: 'University of Central Florida',
+		isAccepted: false,
+		isConfirmed: true,
+		isCheckedIn: true,
+		userLink: 'https://google.com',
+		id: '1',
+		pronouns: {
+			subjective: 'she/her',
+			objective: 'she/her',
+		},
+		phoneNumber: 'num',
+		age: 12,
+		role: 'attendee',
+	};
+	// Remove after adding APIs
+	for (var i = 0; i < 10; i++) {
+		attendeeData.push({
+			firstName: 'Jane Jane',
+			lastName: 'Doe',
+			email: 'email@domain.com',
+			discord: 'discord#1234',
+			oAuth: 'email',
+			school: 'University of Central Florida',
+			isAccepted: false,
+			isConfirmed: true,
+			isCheckedIn: true,
+			userLink: 'https://google.com',
+			id: i + '',
+			pronouns: {
+				subjective: 'she/her',
+				objective: 'she/her',
+			},
+			phoneNumber: 'num',
+			age: 12,
+			role: 'attendee',
+		});
+	}
+
+	return (
+		<>
+			<h1 className=" font-bold text-3xl text-subHeading-color mb-4">
+				{' '}
+				Attendees{' '}
+			</h1>
+			<input
+				id="search"
+				className="mb-8"
+				type={'text'}
+				placeholder="Search Attendees"
+				onChange={(e) => {
+					setSearch(e.target.value);
+				}}
+			/>
+			<table className="w-full border-collapse border-spacing-0">
+				<tbody>
+					<tr>
+						<th>
+							{' '}
```

Comment:
Are these extra space characters in here for a reason? It seems odd to not just include extra space with the text or as a margin if so.

Reply:
Same goes for the attendee row. I'm worried someone's going to come back to it and assume it's garbage and remove the spaces/

Reply:
Ok honestly I have no clue where those comes from. I think it may be my code formatter cause whenever I remove them they readd themselves back? But lemme see if I can turn it off

Reply:
Fixed in other files as well!

Reply:
For future reference, I know that the formatter adds them if I try to add a comment into the middle of JSX, but also probably if there is a random space (rather than a tab) between the tags that then gets formatted on save. Was just curious!